### ISSUE TITLE
On core16, checkbox remote just times out after 600s

### ISSUE THREAD

**Issue by p-gentili** [canonical/checkbox#1341]
### Bug Description

Running latest beta (4.0.0-dev333-1487) on Core16, `checkbox control` command times out after 600s. 

On the same device, same environment, no problems are detected running 3.3.0-dev19-1202.

### To Reproduce

Install latest checkbox beta on the agent side and try to start a remote session via `checkbox-cli control <IP> launcher`.

### Environment

UC16
4.0.0-dev333-1487 (snap)

### Relevant log output

```shell
14:20:04 + checkbox-cli control <IP> checkbox-launcher
14:20:04 Connecting to <IP>:18871. Timeout: 600s
14:29:59 ......................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
14:29:59 Connection timed out.
```
```


### Additional context

http://10.102.156.15:8080/job/cert-tillamook-cascade-kernel-beta/92
